site/content/docs/reference/cli.md

```diff
@@ -10,75 +10,141 @@ menu:
-- `--no-concierge`:
+### Options
 
-  Generate a configuration which does not use the concierge, but sends the credential to the cluster directly
-- `--oidc-ca-bundle strings`:
+```
+      --concierge-api-group-suffix string     Concierge API group suffix (default "pinniped.dev")
+      --concierge-authenticator-name string   Concierge authenticator name
+      --concierge-authenticator-type string   Concierge authenticator type (e.g., 'webhook', 'jwt')
+      --concierge-ca-bundle-data string       CA bundle to use when connecting to the concierge
+      --concierge-endpoint string             API base for the Pinniped concierge endpoint
+      --enable-concierge                      Exchange the token with the Pinniped concierge during login
+  -h, --help                                  help for static
+      --token string                          Static token to present during login
+      --token-env string                      Environment variable containing a static token
+```
 
-  Path to TLS certificate authority bundle (PEM format, optional, can be repeated)
-- `--oidc-client-id string`:
+### SEE ALSO
 
-  OpenID Connect client ID (default: autodiscover) (default "pinniped-cli")
-- `--oidc-issuer string`:
+* [pinniped login]()	 - login
 
-  OpenID Connect issuer URL (default: autodiscover)
-- `--oidc-listen-port uint16`:
+###### Auto generated by spf13/cobra on 22-Mar-2021
+## pinniped version
 
-  TCP port for localhost listener (authorization code flow only)
-- `--oidc-request-audience string`:
+Print the version of this Pinniped CLI
 
-  Request a token with an alternate audience using RFC8693 token exchange
-- `--oidc-scopes strings`:
+```
+pinniped version [flags]
+```
+
+### Options
+
+```
+  -h, --help   help for version
+```
+
+### SEE ALSO
+
+* [pinniped]()	 - pinniped
 
-  OpenID Connect scopes to request during login (default [offline_access,openid,pinniped:request-audience])
-- `--oidc-session-cache string`:
+###### Auto generated by spf13/cobra on 22-Mar-2021
+## pinniped whoami
 
-  Path to OpenID Connect session cache file
-- `--oidc-skip-browser`:
+Print information about the current user
+
+```
+pinniped whoami [flags]
+```
+
+### Options
+
+```
+      --api-group-suffix string     Concierge API group suffix (default "pinniped.dev")
+  -h, --help                        help for whoami
+      --kubeconfig string           Path to kubeconfig file
+      --kubeconfig-context string   Kubeconfig context name (default: current active context)
+  -o, --output string               Output format (e.g., 'yaml', 'json', 'text') (default "text")
+```
 
-  During OpenID Connect login, skip opening the browser (just print the URL)
-- `--static-token string`:
+### SEE ALSO
 
-  Instead of doing an OIDC-based login, specify a static token
-- `--static-token-env string`:
+* [pinniped]()	 - pinniped
 
-  Instead of doing an OIDC-based login, read a static token from the environment
+###### Auto generated by spf13/cobra on 22-Mar-2021
```

Comment:
Maybe we could `grep -v` these lines out?

Reply:
Sure. I thought about that. I didn't do it since I was lazy and wasn't sure it mattered that much. I could see why it might matter from a cosmetic perspective.

Reply:
855686b90ac3779cccfc9f69d338d159ac65fe87